bin/varnishd/cache/cache_wrk.c

```diff
@@ -738,6 +738,28 @@ static struct cli_proto debug_cmds[] = {
 	{ NULL }
 };
 
+void
+WRK_Log(enum VSL_tag_e tag, const char *fmt, ...)
+{
+	struct worker *wrk;
+	va_list ap;
+
+	AN(fmt);
+
+	wrk = THR_GetWorker();
+	CHECK_OBJ_ORNULL(wrk, WORKER_MAGIC);
+
+	if (wrk == NULL)
```

Comment:
discussed during bugwash, this should lead to logging with vxid 0 instead of dropping vsls

Reply:
Done.